server/webhook.go

```diff
@@ -137,7 +138,29 @@ func (wh *webhook) PostNotifications(p *Plugin, instanceID types.ID) ([]*model.P
 		}
 		// If this is a comment-related webhook, we need to check if they have permissions to read that.
 		// Otherwise, check if they can view the issue.
-
+		switch notification.notificationType {
+		case subCommandAssignee:
```

Comment:
I think we could follow the same idea here, having something in common with all these switch case statements we could handle it in the settings and through parameters to the function.

Reply:
Yes, modified this as well.